src/codecs.ts

```diff
@@ -107,12 +151,14 @@ export const DatapointInput = t.intersection([
      * What is the purpose of processing for this datapoint/table?
      *
      * @see https://github.com/transcend-io/privacy-types/blob/main/src/objects.ts
+     * @deprecated - The purpose labels are now associated with fields rather than datapoints to allow for more granularity
```

Comment:
tbh you can delete these, no one is using yet. would just bump the major version